integreat_cms/core/settings.py

```diff
@@ -1011,6 +1011,14 @@
     strtobool(os.environ.get("INTEGREAT_CMS_SUMM_AI_IS_INITIAL", "True"))
 )
 
+# Slugs of regions that prefer Plain German over Easy German in the management command
+SUMM_AI_PLAIN_GERMAN_REGIONS: Final[str | list[str] | None] = [
```

Comment:
```suggestion
SUMM_AI_PLAIN_GERMAN_REGIONS: Final[list[str]] = [
```
This is always a list, right?